GSA_Adapter/GSA_Adapter.csproj

```diff
@@ -111,12 +120,49 @@
   <ItemGroup>
     <Compile Include="AdapterActions\Pull.cs" />
     <Compile Include="AdapterActions\Execute.cs" />
+    <Compile Include="Convert\AdapterID.cs" />
+    <Compile Include="Convert\FromGsa\Elements\Bar.cs" />
+    <Compile Include="Convert\FromGsa\Elements\FEMesh.cs" />
+    <Compile Include="Convert\FromGsa\Properties\LinkConstraint.cs" />
+    <Compile Include="Convert\FromGsa\Loads\Loadcase.cs" />
+    <Compile Include="Convert\FromGsa\Loads\LoadCombination.cs" />
+    <Compile Include="Convert\FromGsa\Properties\Material.cs" />
+    <Compile Include="Convert\FromGsa\Elements\Node.cs" />
+    <Compile Include="Convert\FromGsa\Results\Results.cs" />
+    <Compile Include="Convert\FromGsa\Elements\RigidLink.cs" />
+    <Compile Include="Convert\FromGsa\Properties\SectionProperty.cs" />
+    <Compile Include="Convert\NotSupportedWarning.cs" />
+    <Compile Include="Enums\MaterialType.cs" />
+    <Compile Include="Convert\FromGsa\Properties\SurfaceProperty.cs" />
+    <Compile Include="Convert\ToGsa\Elements\Bar.cs" />
+    <Compile Include="Convert\ToGsa\Elements\FEMesh.cs" />
+    <Compile Include="Convert\ToGsa\Group.cs" />
+    <Compile Include="Convert\ToGsa\Loads\AreaLoads.cs" />
+    <Compile Include="Convert\ToGsa\Loads\BarLoads.cs" />
+    <Compile Include="Convert\ToGsa\Loads\Factor.cs" />
+    <Compile Include="Convert\ToGsa\Loads\ForceType.cs" />
+    <Compile Include="Convert\ToGsa\Loads\Load.cs" />
+    <Compile Include="Convert\ToGsa\Loads\Loadcase.cs" />
+    <Compile Include="Convert\ToGsa\Loads\LoadCombination.cs" />
+    <Compile Include="Convert\ToGsa\Loads\NodeLoad.cs" />
+    <Compile Include="Convert\ToGsa\Loads\RotationVector.cs" />
+    <Compile Include="Convert\ToGsa\Loads\TranslationVector.cs" />
+    <Compile Include="Convert\ToGsa\Properties\LinkConstraint.cs" />
+    <Compile Include="Convert\ToGsa\Properties\Material.cs" />
+    <Compile Include="Convert\ToGsa\Elements\Node.cs" />
+    <Compile Include="Convert\ToGsa\Elements\RigidLink.cs" />
+    <Compile Include="Convert\ToGsa\Properties\SectionProperty.cs" />
+    <Compile Include="Convert\ToGsa\Properties\SurfaceProperties.cs" />
+    <Compile Include="Convert\ToGsa\_ToGsa.cs" />
     <Compile Include="CRUD\ReadResults.cs" />
     <Compile Include="CRUD\ReadResultsElements.cs" />
     <Compile Include="CRUD\ReadResultsGlobal.cs" />
     <Compile Include="CRUD\Update.cs" />
     <Compile Include="CRUD\UpdateProperty.cs" />
     <Compile Include="CRUD\Delete.cs" />
+    <Compile Include="Enums\Output_Axis.cs" />
+    <Compile Include="Enums\Output_Init_Flags.cs" />
+    <Compile Include="Enums\UnitType.cs" />
```

Comment:
While I understand your point of view, I still don't like the the fact we're having an Enum folder in the Adapter.
Furthermore, GSA_oM has already an `Enums.cs` file. I would:
- Create an `Enums` folder in the GSA_oM
- Move the `Enums.cs` file in there, maybe splitting each Enum in a separate file (for consistency with the rest, but not important anyway).
- Move the `Enums` out of the Adapter into this new folder.

At that point, simply adding a `using BH.oM.GSA;` in the Convert folder files would make them compile.

Reply:
Would not want to move these to the oM for the simple reason that they are only here to help with the inner workings of the adapter. There is literally 0 use for users to be able to create these, so should not sit where they are exposed.

Reason we have them defined here is that we can not simply acess them from the GSA api, but still need them for various functions to work (or to avoid having to use confusing ints as arguments).

So even do I understand your arguments, these need to sit in the adapter for reasons above, and IMO, I think they sit better where they are now than hidden in the convert folder (even ofc this is just a subjective opinion).